test-data/unit/stubgen.test

```diff
@@ -367,19 +413,19 @@ from re import match, search, sub
 __all__ = ['match', 'sub', 'x']
 x = 1
 [out]
-from re import match as match, sub as sub
```

Comment:
Don't forget about fixing this as discussed.

Reply:
Fixed